pages/index.vue

```diff
@@ -28,19 +28,23 @@
       </v-container>
     </v-parallax>
     <VVAnnouncements v-if="announcements && announcements.length" :announcements="announcements"/>
+    <VVMeetups title="Meetups" loadingMessage="Checking for meetups..." laterMessage="No meetups found, check back later?"/>
```

Comment:
**praise**: Very nice work with error and loading cases

Reply:
Thanks :D